{{cookiecutter.project_slug}}/frontend/.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
**question** Is this pre-commit hook something that can be included in https://github.com/sixfeetup/cookiecutter-sixiedjango/pull/29 ?

Reply:
Well what husky does is just pausing commit until npm lint-staged command is working well and if that fails rejects the commit. So i think it looks like we can do same in the PR that u shared. So we could give it a try and see if it works well then we could remove these from FE

Reply:
I'd like to have husky for nodejs and pre-commit for the backend. FE devs are familiar with husky and shouldn't have to learn a new tool to get the job done.